react/src/pages/ServingListPage.tsx

```diff
@@ -288,6 +288,16 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
                     <Typography.Text code>{endpoint_id}</Typography.Text>
                   ),
                 },
+                {
+                  title: t('modelService.ServiceEndpoint'),
+                  dataIndex: 'endpoint_id',
+                  render: (endpoint_id, row) =>
+                    row.url ? (
+                      <Typography.Text copyable>{row.url}</Typography.Text>
```

Comment:
It's better to make a link
```suggestion
                      <Typography.Link copyable href={row.url} target="_blank">
                        {row.url}
                      </Typography.Link>
```